ballerina-to-openapi/src/main/java/io/ballerina/openapi/service/mapper/model/OperationBuilder.java

```diff
@@ -0,0 +1,109 @@
+/*
+ *  Copyright (c) 2023, WSO2 LLC. (http://www.wso2.org) All Rights Reserved.
+ *
+ *  WSO2 LLC. licenses this file to you under the Apache License,
+ *  Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing,
+ *  software distributed under the License is distributed on an
+ *  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ *  KIND, either express or implied.  See the License for the
+ *  specific language governing permissions and limitations
+ *  under the License.
+ */
+package io.ballerina.openapi.service.mapper.model;
+
+import io.swagger.v3.oas.models.Components;
+import io.swagger.v3.oas.models.Operation;
+import io.swagger.v3.oas.models.parameters.Parameter;
+import io.swagger.v3.oas.models.parameters.RequestBody;
+import io.swagger.v3.oas.models.responses.ApiResponses;
+
+import java.util.Objects;
+
+import static io.ballerina.openapi.service.mapper.Constants.SLASH;
+
+/**
+ * This class will hold operation details specific to HTTP operation.
+ */
+public class OperationBuilder {
```

Comment:
I think naming the model "OperationMetadataHolder" would be appropriate since its purpose is to store metadata related to operations used on resources. This is because, in its usage, we won't retrieve all this data in one place. WDYT?

Reply:
As we discussed with offline let's add the name `OperationInventory` :)

Reply:
We have agreed to use `OperationInventory`